### ISSUE TITLE
As a privileged Rat I want to be able to restore previous versions of paperwork

### ISSUE THREAD

**Issue by UncleClapton** [FuelRats/fuelrats.com#347]
## Acceptance Criteria

- [ ] 

## Additional Information

Blocked by: [API-398](https://jira.fuelrats.com/browse/API-398)

**Comment by SuperManifolds** [comment]
Won't do, too technically complex
